hugo/content/Upgrade/upgrade35to4.md

```diff
@@ -18,11 +18,11 @@ As with any upgrade, please ensure you have taken recent backups of all relevant
 ##### Prerequisites.
 You will need the following items to complete the upgrade:
 
-* The latest PGO 4.1 code for the Postgres Operator available
-* The latest PGO 4.1 client binary
+* The latest PGO 4.2.0 code for the Postgres Operator available
+* The latest PGO 4.2.0 client binary
 
 ##### Step 0
-Create a new Centos/Redhat user with the same permissions are the existing user used to install the Crunchy Postgres Operator. This is necessary to avoid any issues with environment variable differences between 3.5 and 4.1.
+Create a new Centos/Redhat user with the same permissions are the existing user used to install the Crunchy Postgres Operator. This is necessary to avoid any issues with environment variable differences between 3.5 and 4.2.0.
```

Comment:
sp. `CentOS` and `Red Hat`. We call this a `Linux user` later. Can we use it here, too?

```suggestion
Create a new Linux user with the same permissions are the existing user used to install the Crunchy Postgres Operator. This is necessary to avoid any issues with environment variable differences between 3.5 and 4.2.0.
```

Reply:
Yes, I think it's fair to use Linux user in both places given that this is referring to an upgrade of an existing environment. I'll make that change.